plugins/org.eclipse.reddeer.workbench/src/org/eclipse/reddeer/workbench/impl/editor/AbstractEditor.java

```diff
@@ -122,7 +123,14 @@ public boolean isDirty() {
 	public void save() {
 		activate();
 		log.info("Save editor");
-		new ShellMenuItem(new WorkbenchShell(), "File", "Save").select();
+		try {
+			new ShellMenuItem(new WorkbenchShell(), "File", "Save").select();
```

Comment:
it would be a lot safer if you call ShellMenuItem.isEnabled -> if false throw ex.